servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingStreamingHttpRequest.java

```diff
@@ -175,7 +175,14 @@ default InputStream payloadBodyInputStream() {
 
     /**
      * Returns a {@link BlockingStreamingHttpRequest} with its underlying payload transformed to {@link Buffer}s,
-     * with access to the trailers.
+     * with access to the <a href="https://tools.ietf.org/html/rfc7230#section-4.1.2">trailer</a>s.
+     * <p>
+     * Trailers are supported only when the {@link HttpProtocolVersion#HTTP_1_1 HTTP/1.1} version is used or above.
+     * HTTP/1.1 uses <a href="https://tools.ietf.org/html/rfc7230#section-4.1">Chunked Transfer Coding</a> to deliver
```

Comment:
```java
In order to use trailers with HTTP/1.1, you must use
<a href="https://tools.ietf.org/html/rfc7230#section-4.1">Chunked Transfer Coding</a>.
This method will not modify existing headers so if you intend to add trailers
to the request where non may have previously existed you should do the following:
<ul>
<li>Remove the {@link HttpHeaderNames#CONTENT_LENGTH} header</li>
<li>(optional) Add the {@link HttpHeaderNames#TRANSFER_ENCODING}</li>
<li>(optional) Consider putting the names of each trailer in the {@link HttpHeaderNames#TRAILERS} header</li>
<li>(optional) Validating your peer accepts trailers via the {@link HttpHeaderNames#TE} header</li>
</ul>
```

Reply:
I've removed these comments because the behavior was reverted.